config.yml

```diff
@@ -1,4 +1,4 @@
-traceability_export: build/doc/${PRODUCT}/traceability_export.json
+traceability_export: traceability_export.json
```

Comment:
`./traceability_export.json` clarifies that a relative path is supported. Is this configuration value necessary? It's also defined as an environment variable in https://github.com/melexis/traceability-viewer/pull/2/commits/a3a2399ef8198823868b898ad42d20bd6660e0c4#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8cR1

Reply:
I think I can remove that part because it is copied in the docker container.